yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java

```diff
@@ -87,6 +96,13 @@ public Date getDateOfBirth() {
         return wrapped.getAttribute(ATTRIBUTE_DOB, Date.class);
     }
 
+    @Override
+    public Boolean isAgeVerified() {
+        Boolean isAgeOver = wrapped.getAttribute(ATTRIBUTE_AGE_OVER, Boolean.class);
```

Comment:
For what I can see, this cannot work.
The attribute starts with `age_over:` or `age_under:` (e.g. `age_over:18`)

I believe you need changes in the implementation of Profile where you can query for an `age_under:*` or `age_under:*` and return one if available.

Reply:
Yep, see on this page `https://lampkicking.atlassian.net/wiki/spaces/IPASS/pages/29556930/Attribute+and+backing+document+names`

Reply:
Fixed